lib/pages/add_task.dart

```diff
@@ -16,43 +18,46 @@ class _AddTaskScreenState extends State<AddTaskScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      backgroundColor: const Color.fromARGB(255, 48, 48, 48),
+      bottomNavigationBar: const BottomNav(),
+      backgroundColor: bgColor,
       appBar: AppBar(
-        title: const Text(
+        title: Text(
           'Add a Task',
-          style: TextStyle(letterSpacing: 2.0, color: Colors.white),
+          style: TextStyle(letterSpacing: 2.0, color: white),
         ),
         centerTitle: true,
-        backgroundColor: Colors.black,
+        backgroundColor: black,
       ),
       body: Column(
         children: [
           Padding(
             padding: const EdgeInsets.all(8.0),
             child: TextField(
+              key: const Key('Add a task text field'),
               controller: _addTask,
-              decoration: const InputDecoration(
+              decoration: InputDecoration(
                 filled: true,
-                fillColor: Color.fromARGB(255, 255, 255, 255),
+                fillColor: white,
                 hintText: 'Add a Task here',
               ),
             ),
           ),
           ElevatedButton.icon(
-              icon: const Icon(
+              key: const Key('Add Task Button'),
+              icon: Icon(
                 Icons.add,
-                color: Colors.white,
+                color: white,
               ),
               onPressed: () {
                 context
                     .read<TaskCubit>()
-                    .addTask(taskName: Task(id: null, name: _addTask.text));
+                    .addTask(task: Task(name: _addTask.text));
                 _addTask.clear();
                 context.go('/');
               },
-              label: const Text('Add Task',
-                  style: TextStyle(letterSpacing: 2.0, color: Colors.white)),
-              style: ElevatedButton.styleFrom(backgroundColor: Colors.black)),
+              label: Text('Add Task',
+                  style: TextStyle(letterSpacing: 2.0, color: white)),
```

Comment:
This is totally fine. The indentation looks weird to me personally, but no pressure to change it. Just a tip in case the indentation ever looks weird to you, try adding commas after the parameters or parentheses to see what the formatter does. It puts a line break at each comma.

Reply:
Got it